Capri.Web/Configuration/Mapper/EnumMappingProfile.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.ComponentModel;
+using AutoMapper;
+using Capri.Database.Entities;
+
+namespace Capri.Web.Configuration.Mapper
+{
+    public class EnumMappingProfile : Profile
```

Comment:
`EnumMappingProfile`? Does it map all enums in system? Including future ones?

Reply:
Yes, it does

Reply:
Ok let it stay like that.